CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Removed
 - Disable `rpc-caller-procedure` header temporarily by stop propoagating CallerProcedure in the outbound middleware.
+### Added
+- gRPC: log header values upon validation error
```

Comment:
Should we put this in the backlog of 1.53.0 actually?

Reply:
IMO it's better we maintain clean change-log by keeping this part of 1.53.2 release

Reply:
Oh it was actually added after 1.53.0, so all good